Library/src/Systems/Maintenance/MaintenanceSystem.cpp

```diff
@@ -45,27 +47,32 @@ MaintenanceSystem::~MaintenanceSystem()
 
 void MaintenanceSystem::GetMaintenanceInfo(MaintenanceInfoCallback Callback)
 {
-	const MaintenanceInfoCallback GetMaintenanceInfoCallback = [Callback](const MaintenanceInfoResult& Result)
+	if (Running == true)
```

Comment:
Could we rename `Running` to be something that helps the reader understand it's purpose a little clearer? I'm imaging something like `AllowMaintenanceInfoRequests`

Reply:
That's reasonable.